scrapper/src/main/java/edu/java/httpclient/StackOverflowClient.java

```diff
@@ -0,0 +1,45 @@
+package edu.java.httpclient;
+
+import edu.java.dto.httpclient.StackExchangeQuestionResponse;
+import edu.java.dto.httpclient.StackExchangeResponse;
+import java.util.HashMap;
+import java.util.Map;
+import org.springframework.web.reactive.function.client.WebClient;
+import org.springframework.web.reactive.function.client.support.WebClientAdapter;
+import org.springframework.web.service.invoker.HttpServiceProxyFactory;
+
+public class StackOverflowClient {
+    private final StackOverflowHttpExchange stackOverflowHttpExchange;
+    private final static String BASEURL = "https://api.stackexchange.com/";
+    private final static String API_VERSION = "2.3";
+    private final static Map<String, String> PARAMS = new HashMap<>();
```

Comment:
`private final static Map<String, String> PARAMS = Map.of("site", "stackoverflow.com");`